src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -70,6 +70,10 @@ class AdamsBashforthN : public LtsTimeStepper::Inherit {
                 gsl::not_null<History<Vars, DerivVars>*> history,
                 const TimeDelta& time_step) const noexcept;
 
+  template <typename Vars, typename DerivVars>
+  Vars dense_output(const History<Vars, DerivVars>& history,
```

Comment:
Should this have a doxygen comment? I'm not sure if
1. it needs it for clarity (can we assume people know what dense output is?)
2. it is meant to be used by people not familiar with the time steppers

Reply:
It's documented in the base class.  (But this did remind me that I'd forgotten to include information on call sequencing there, so you should look at that again.)